### ISSUE TITLE
Source Zendesk Chat: Internal Server Error: For input string ending in 0

### ISSUE THREAD

**Issue by gustavohwulee** [airbytehq/airbyte#23833]
## Environment
- **Airbyte version**:  0.40.30
- **OS Version / Instance**: Ubuntu 18.04 AWS EC2
- **Deployment**: Docker
- **Source Connector and version**: source-zendesk-chat (0.1.13)
- **Destination Connector and version**: airbyte/destination-s3 (0.3.20)
- **Step where error happened**: Setup new connection

## Current Behavior
![image](https://user-images.githubusercontent.com/38598129/223495548-4910e3c1-a896-43fc-a590-a89ba434e749.png)

## Expected Behavior
connection to be setup

## Logs
```
{"message":"Internal Server Error: For input string: \"1000000000000000000000000000000000\"","exceptionClassName":"java.lang.NumberFormatException","exceptionStack":["java.lang.NumberFormatException: For input string: \"1000000000000000000000000000000000\"","\tat java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:67)","\tat java.base/java.lang.Long.parseLong(Long.java:708)","\tat java.base/java.lang.Long.valueOf(Long.java:1156)","\tat org.jooq.tools.json.Yylex.yylex(Yylex.java:659)","\tat org.jooq.tools.json.JSONParser.nextToken(JSONParser.java:263)","\tat org.jooq.tools.json.JSONParser.parse(JSONParser.java:112)","\tat org.jooq.tools.json.JSONParser.parse(JSONParser.java:75)","\tat org.jooq.tools.json.JSONParser.parse(JSONParser.java:69)","\tat org.jooq.JSONB.parsed(JSONB.java:104)","\tat org.jooq.JSONB.toString(JSONB.java:131)","\tat io.airbyte.config.persistence.DbConverter.buildActorCatalog(DbConverter.java:229)","\tat java.base/java.util.Optional.map(Optional.java:260)","\tat io.airbyte.config.persistence.ConfigRepository.getActorCatalog(ConfigRepository.java:1341)","\tat io.airbyte.commons.server.handlers.SchedulerHandler.discoverSchemaForSourceFromSourceId(SchedulerHandler.java:260)","\tat io.airbyte.server.apis.SourceApiController.lambda$discoverSchemaForSource$5(SourceApiController.java:91)","\tat io.airbyte.server.apis.ApiHelper.execute(ApiHelper.java:18)","\tat io.airbyte.server.apis.SourceApiController.discoverSchemaForSource(SourceApiController.java:91)","\tat io.airbyte.server.apis.$SourceApiController$Definition$Exec.dispatch(Unknown Source)","\tat io.micronaut.context.AbstractExecutableMethodsDefinition$DispatchedExecutableMethod.invoke(AbstractExecutableMethodsDefinition.java:378)","\tat io.micronaut.context.DefaultBeanContext$4.invoke(DefaultBeanContext.java:594)","\tat io.micronaut.web.router.AbstractRouteMatch.execute(AbstractRouteMatch.java:303)","\tat io.micronaut.web.router.RouteMatch.execute(RouteMatch.java:111)","\tat io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:103)","\tat io.micronaut.http.server.RouteExecutor.lambda$executeRoute$14(RouteExecutor.java:659)","\tat reactor.core.publisher.FluxDeferContextual.subscribe(FluxDeferContextual.java:49)","\tat reactor.core.publisher.Flux.subscribe(Flux.java:8660)","\tat reactor.core.publisher.FluxFlatMap$FlatMapMain.onNext(FluxFlatMap.java:426)","\tat io.micronaut.reactive.reactor.instrument.ReactorSubscriber.onNext(ReactorSubscriber.java:57)","\tat reactor.core.publisher.MonoCreate$DefaultMonoSink.success(MonoCreate.java:172)","\tat io.micronaut.http.server.netty.RoutingInBoundHandler$4.doOnComplete(RoutingInBoundHandler.java:965)","\tat io.micronaut.core.async.subscriber.CompletionAwareSubscriber.onComplete(CompletionAwareSubscriber.java:79)","\tat io.micronaut.http.server.netty.jackson.JsonContentProcessor$1.doOnComplete(JsonContentProcessor.java:136)","\tat io.micronaut.core.async.subscriber.CompletionAwareSubscriber.onComplete(CompletionAwareSubscriber.java:79)","\tat java.base/java.util.Optional.ifPresent(Optional.java:178)","\tat io.micronaut.core.async.processor.SingleThreadedBufferingProcessor.doOnComplete(SingleThreadedBufferingProcessor.java:48)","\tat io.micronaut.jackson.core.parser.JacksonCoreProcessor.doOnComplete(JacksonCoreProcessor.java:94)","\tat io.micronaut.core.async.subscriber.SingleThreadedBufferingSubscriber.onComplete(SingleThreadedBufferingSubscriber.java:71)","\tat io.micronaut.http.server.netty.jackson.JsonContentProcessor.doOnComplete(JsonContentProcessor.java:161)","\tat io.micronaut.core.async.subscriber.CompletionAwareSubscriber.onComplete(CompletionAwareSubscriber.java:79)","\tat io.micronaut.http.netty.reactive.HandlerPublisher.publishMessage(HandlerPublisher.java:383)","\tat io.micronaut.http.netty.reactive.HandlerPublisher.flushBuffer(HandlerPublisher.java:470)","\tat io.micronaut.http.netty.reactive.HandlerPublisher.publishMessageLater(HandlerPublisher.java:360)","\tat io.micronaut.http.netty.reactive.HandlerPublisher.complete(HandlerPublisher.java:423)","\tat io.micronaut.http.netty.reactive.HandlerPublisher.handlerRemoved(HandlerPublisher.java:418)","\tat io.netty.channel.AbstractChannelHandlerContext.callHandlerRemoved(AbstractChannelHandlerContext.java:1122)","\tat io.netty.channel.DefaultChannelPipeline.callHandlerRemoved0(DefaultChannelPipeline.java:637)","\tat io.netty.channel.DefaultChannelPipeline.remove(DefaultChannelPipeline.java:477)","\tat io.netty.channel.DefaultChannelPipeline.remove(DefaultChannelPipeline.java:423)","\tat io.micronaut.http.netty.stream.HttpStreamsHandler.removeHandlerIfActive(HttpStreamsHandler.java:483)","\tat io.micronaut.http.netty.stream.HttpStreamsHandler.handleReadHttpContent(HttpStreamsHandler.java:319)","\tat io.micronaut.http.netty.stream.HttpStreamsHandler.channelRead(HttpStreamsHandler.java:282)","\tat io.micronaut.http.netty.stream.HttpStreamsServerHandler.channelRead(HttpStreamsServerHandler.java:134)","\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)","\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)","\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)","\tat io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:93)","\tat io.netty.handler.codec.http.websocketx.extensions.WebSocketServerExtensionHandler.channelRead(WebSocketServerExtensionHandler.java:99)","\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)","\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)","\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)","\tat io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)","\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)","\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)","\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)","\tat io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)","\tat io.netty.handler.codec.MessageToMessageCodec.channelRead(MessageToMessageCodec.java:111)","\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)","\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)","\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)","\tat io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:93)","\tat io.netty.handler.codec.http.HttpServerKeepAliveHandler.channelRead(HttpServerKeepAliveHandler.java:64)","\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)","\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)","\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)","\tat io.netty.handler.flow.FlowControlHandler.dequeue(FlowControlHandler.java:200)","\tat io.netty.handler.flow.FlowControlHandler.read(FlowControlHandler.java:139)","\tat io.netty.channel.AbstractChannelHandlerContext.invokeRead(AbstractChannelHandlerContext.java:837)","\tat io.netty.channel.AbstractChannelHandlerContext.read(AbstractChannelHandlerContext.java:814)","\tat io.micronaut.http.netty.reactive.HandlerPublisher.requestDemand(HandlerPublisher.java:165)","\tat io.micronaut.http.netty.stream.HttpStreamsHandler$2.requestDemand(HttpStreamsHandler.java:273)","\tat io.micronaut.http.netty.reactive.HandlerPublisher$ChannelSubscription.receivedDemand(HandlerPublisher.java:556)","\tat io.micronaut.http.netty.reactive.HandlerPublisher$ChannelSubscription.lambda$request$0(HandlerPublisher.java:494)","\tat io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)","\tat io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)","\tat io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)","\tat io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:566)","\tat io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)","\tat io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)","\tat io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)","\tat java.base/java.lang.Thread.run(Thread.java:1589)"]}
```

## Steps to Reproduce
1. Create a Zendesk Source
2. Create a s3 Destination
3. Try to set a new connection and it will fail on the discover_schema
4. Not sure if a problem specific to our data

## Are you willing to submit a PR?
yes

**Comment by natalyjazzviolin** [comment]
@gustavohwulee please upgrade your Airbyte instance and run this connection again.

Might be related to https://github.com/airbytehq/airbyte/issues/13651

**Comment by RobertoBonnet** [comment]
Same problem here with airbyte 0.42.0. From Zendesk Chat to BigQuery

**Comment by leo-schick** [comment]
I found the reason for this issue and fixed it in PR #24190:

The source Zendesk Chat uses in stream `accounts` a minimum/maximum definition of size from `-1e33` to `1e33` for a specific property. This property (`/plans/price`) is not even specified in the Zendesk Chat API documentation [here](https://developer.zendesk.com/api-reference/live-chat/chat-api/accounts/). So in my PR, I remove the minimum/maximum definition for this property to fix this. (Schema change / might be a breaking change for existing users).

The strange thing is, that this error happend suddenly for me: I used a Airbyte version 0.40.xx (I guess it was 0.40.27) and I was able to setup the Zendesk Chat stream. But since my upgrade to 0.41.0, it broke and I see now (without this fix) the setup in a timeout:

<details><summary><strong>Screenshots</strong></summary>

![image](https://user-images.githubusercontent.com/67712864/225934103-b51a516c-489c-46a9-b44a-b15361a36d71.png)

![image](https://user-images.githubusercontent.com/67712864/225934120-7c380f81-5931-4a64-a749-343f5c2b3c20.png)

![image](https://user-images.githubusercontent.com/67712864/225934142-5fd56b40-b70a-44f0-b4a5-541a588c7123.png)

![image](https://user-images.githubusercontent.com/67712864/225934156-4fb72070-3992-4c88-bfa1-714133b5b285.png)

</details>

I would have expected a nicer error message here...

When it comes to the `1e33` isse: According to [RFC 71519](https://www.rfc-editor.org/rfc/rfc7159#section-6), the number `1e33` is too big for a JSON number. Maybe someone from the Airbyters could put this into a unit test for `discover` so that this issue get caught up during connector development time.
Take a look at [Stackoverflow: JSON integers: limit on size](https://stackoverflow.com/questions/13502398/json-integers-limit-on-size) also.

**Comment by leo-schick** [comment]
This is a bit out-of-scope for this issue but I want to mention it here because it is connected to this error for me:

After upgrading to the new airbyte version I am not able to do anything with the connection which I had set up before the upgrade of Airbyte:

![image](https://user-images.githubusercontent.com/67712864/225943414-1e2c7460-2899-4ac1-b9b9-b41aa6d17019.png)

When I click on the red marked areas, I always end up in a "Unknown error occurred".

![image](https://user-images.githubusercontent.com/67712864/225943715-0b16bbba-4828-41a4-8867-bff08b9143cc.png)

Server logs do not show any error. (I don't know if there is a kind of verbose log to see more in the logs...)

Now I am stuck with two connections in airbyte: The new one which works fine after the patch (PR #24190) and the second which existed before which I cannot delete from the UI. Thankfully, I was able to disable it from cron via the UI ... but would like to get rid of it which isn't possible with the default UI commands.
A delete button would be great in such cases ...
